modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/processor/stream/function/StreamFunctionProcessor.java

```diff
@@ -42,6 +44,7 @@ protected void processEventChunk(ComplexEventChunk complexEventChunk, Processor
                     }
                     outputData = process(inputData);
                     complexEventPopulater.populateComplexEvent(complexEvent, outputData);
+                    break;
```

Comment:
Keeping it makes no harm. But technically it is not needed. +1 to remove if there is no other compelling reason.